src/Promact.TestCaseManagement.DomainModel/Enums/TeamRole.cs

```diff
@@ -5,4 +5,4 @@ public enum TeamRole
         TeamLeader,
```

Comment:
TeamRole should not be enum. Role management is not part of test-case-management. It will come from oauth-server. should be handled using strings.